packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -796,6 +796,17 @@ describe('build', () => {
     expect(Object.keys(env).includes('VERCEL_ANALYTICS_ID')).toEqual(true);
   });
 
+  it('should not set `VERCEL_ANALYTICS_ID` if new @vercel/speed-insights package is present', async () => {
+    const cwd = fixture('vercel-speed-insights');
+    const output = join(cwd, '.vercel/output');
+    client.cwd = cwd;
+    const exitCode = await build(client);
+    expect(exitCode).toEqual(0);
+
+    const env = await fs.readJSON(join(output, 'static', 'env.json'));
+    expect(Object.keys(env).includes('VERCEL_ANALYTICS_ID')).toEqual(false);
```

Comment:
**suggestion:** you may turn this [into](https://jestjs.io/docs/expect#tohavepropertykeypath-value) `expect(env).not.toHaveProperty('VERCEL_ANALYTICS_ID')`

Reply:
@chriswdmr This appears to be failing right now: https://github.com/vercel/vercel/actions/runs/8377067208/job/22946124053?pr=11305